### PR TITLE
Recipe model

### PR DESCRIPTION
**Pull request by KaskMIL**
# In this new Pull Request I'm adding the following features:
- Create a migration for Recipes referencing Users.
- Create a migration Recipes_food referencing Recipes and Foods.
- Create a model Recipe that belong to User.
- Create a model FoodRecipe that belong to Recipe and Food.
- Create controllers for Recipe and Recipe_Food.
- Create helpers for Recipe and Recipe_Food.

### CODE REVIEW

File: app/models/recipe_food.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class RecipeFood < ApplicationRecord
+  belongs_to :foods
+  belongs_to :recipes
+end
```
Comment (worashf) [inline app/models/recipe_food.rb:4]: It is better to use the rails naming convention for defining belongs to references, like using the singular model name for belongs to reference. Like 
    belongs_to :food
  belongs_to :recipe

File: db/migrate/20221004130844_create_recipes.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreateRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipes do |t|
+      t.string :name
+      t.integer :preparationTime
+      t.integer :cookingTime
```
Comment (worashf) [inline db/migrate/20221004130844_create_recipes.rb:6]: It is better to use under scorer (_) separated column names, like preparation_time and cooking_time.
It is better to add other migration to rename this column.
You can follow this example
rails g migration rename_preparationTime_to_preparation_time
class RenamePreparationTimeToPreparationTime < ActiveRecord::Migration
  def change
  end
end

Modify it as follows:

class RenamePreparationToPreparationTime < ActiveRecord::Migration
  def change
    rename_column :recipes, :preparationTime, :preparation_time
  end
end

**Review by worashf — CHANGES_REQUESTED** [review]
### Hello @KaskMIL :hand: 
You are doing good so far, Your project has some changes to be made to make it complete.
### 
  Hilights :rocket: 
:heavy_check_mark:  No linters error
:heavy_check_mark:  Your commit messages are professional
:heavy_check_mark: Good  feature implementation

Change Required :recycle: 
 Your project is almost there, and you can do it

**Comment by KaskMIL** [comment]
Hi @worashf 
Thanks for your review, I applied the changes suggested.
Cheers

**Review by worashf — APPROVED** [review]
Hello @KaskMIL :hand: 
You did a good job so far. Your project is complete there is nothing to say it is time to merge.
### 
Highlights :rocket: 
:heavy_check_mark:  Implemented the changes correctly
:heavy_check_mark:  No linters error
:heavy_check_mark: Your commit message is clear.
### 
 Project Status: Approved :heavy_check_mark: 
cheers and happy coding :wine_glass:  

Optional suggestions



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**